vault.go

```diff
@@ -0,0 +1,234 @@
+package main
```

Comment:
Ok, this worked for me.

Diff for `data.go`:
```diff
diff --git a/data.go b/data.go
index 0776b65..32486fd 100644
--- a/data.go
+++ b/data.go
@@ -14,6 +14,7 @@ import (
        "time"
 
        "github.com/blang/vfs"
+       "github.com/hairyhenderson/gomplate/vault"
 )
 
 // logFatal is defined so log.Fatal calls can be overridden for testing
@@ -83,7 +84,7 @@ type Source struct {
        Params map[string]string
        FS     vfs.Filesystem // used for file: URLs, nil otherwise
        HC     *http.Client   // used for http[s]: URLs, nil otherwise
-       VC     *Vault         // used for vault: URLs, nil otherwise
+       VC     *vault.Vault   // used for vault: URLs, nil otherwise
        Header http.Header    // used for http[s]: URLs, nil otherwise
 }
 
@@ -303,7 +304,7 @@ func readHTTP(source *Source, args ...string) ([]byte, error) {
 
 func readVault(source *Source, args ...string) ([]byte, error) {
        if source.VC == nil {
-               source.VC = NewVault()
+               source.VC = vault.NewVault()
                source.VC.Login()
                addCleanupHook(source.VC.Logout)
        }
```

Diff for `vault.go`:
```diff
diff vault.go vault/vault.go 
1c1
< package main
---
> package vault
```